package.json

```diff
@@ -48,6 +52,7 @@
     "eslint-plugin-react": "^7.31.11",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-watch": "^8.0.0",
+    "openapi-typescript": "^6.1.0",
```

Comment:
this is dev dependency

Reply:
Changed in 8eac28f

Reply:
this is still inside dependencies. Should be inside devDependencies

Reply:
![image](https://user-images.githubusercontent.com/72614768/219842794-c0fa89a8-25a6-454e-bf40-2bb288ba79d0.png)
It is in devDependencies now.